Modules/IO/NRRD/test/itkNrrdImageIOTest.h

```diff
@@ -99,6 +100,20 @@ itkNrrdImageIOTestReadWriteTest(std::string fn, unsigned int size, std::string i
   {
     // Generate a random image.
     image = itkNrrdImageIOTestGenerateRandomImage<TPixelType, VImageDimension>(size);
+
+    // add custom metadata
+    itk::MetaDataDictionary dictionary;
+
+    itk::EncapsulateMetaData(dictionary, "ASimpleString", std::string("a string"));
+    itk::EncapsulateMetaData(dictionary, "ASimpleFloat", 1.2f);
+    itk::EncapsulateMetaData(dictionary, "ASimpleDouble", 2.3);
+    itk::EncapsulateMetaData(dictionary, "ASimpleInt", 3);
+
+    itk::EncapsulateMetaData(dictionary, "Array<int>", itk::Array<int>(3, 1));
+    itk::EncapsulateMetaData(dictionary, "Array<float>", itk::Array<float>(4, 2.2f));
+    itk::EncapsulateMetaData(dictionary, "Array<double>", itk::Array<double>(5, 3.3d));
```

Comment:
In C++, floating point numbers are double, unless otherwise specified (e.g. by `f` suffix). Visual Studio errors out: `D:\a\1\s\Modules\IO\NRRD\test\itkNrrdImageIOTest.h(114): error C3688: invalid literal suffix 'd'; literal operator or literal operator template 'operator ""d' not found`

Reply:
i fixed this in the last force-push

Reply:
You fixed it for the non-array variant, a few lines above (line 109).